docs/who-uses-meson-python.rst

```diff
@@ -0,0 +1,34 @@
+.. SPDX-FileCopyrightText: 2023 The meson-python developers
+..
+.. SPDX-License-Identifier: MIT
+
+.. _who-uses-meson-python:
+
+**************************
+Who uses ``meson-python``?
+**************************
+
+Here's a curated list of projects using ``meson-python``.
+
+
+.. list-table::
+   :widths: 20 80
+
+   * - `SciPy <https://github.com/scipy/scipy>`_
+     - Probably the most complex project using Meson and ``meson-python``.
+       Staring at the build definition too long may cause headaches.
```

Comment:
I'm not sure if I should confirm that this is true, or be slightly offended:)

Reply:
Anyone who gets headaches from looking at SciPy's build definition should definitely not look at the systemd build definition, is all I can say. :)

Reply:
I didn't expect it to be taken seriously when I wrote that sentence. Unless everyone is happy with the pun, I would replace with something a bit more useful. Something like: "It combines CPython extensions and libraries written in C, C++, Fortran, Cython, Pythran, different ASM variants, ..." maybe something about CPU feature detection or BLAS libraries. I'm sure @rgommers can suggest something.